core/src/create.jl

```diff
@@ -173,14 +187,27 @@ function Control(db::DB, config::Config)::Control
         condition_value,
         control_state,
         logic_mapping,
-        graph,
         record,
     )
 end
 
-function Parameters(db::DB, config::Config)::Parameters
+function PidControl(db::DB, config::Config)::PidControl
+    static = load_structvector(db, config, PidControlStaticV1)
+
+    proportional = coalesce.(static.proportional, NaN)
```

Comment:
Not for in this PR, but I'm wondering whether we should move this to `load_structvector`. We probably need a separate issue to test our handling of missing data in tables.